.github/workflows/java-build-cpp.yml

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+name: Java Bindings - C++ Build
+
+on:
+  workflow_dispatch: {}
+  schedule:
+    - cron: 38 8 * * *
```

Comment:
Could you receive the ci nitification if it failed

Reply:
These badges in `java/README.md` are for tracking the CI results (they may look broken now because the PR haven't merged):

https://github.com/zhztheplayer/velox/blob/1134924aff5e67ecc59844245c9f3d3756c021f0/java/README.md?plain=1#L3-L4

Another way is to find the results in the actions page: https://github.com/facebookincubator/velox/actions.

By the way, I made the C++ build test daily because I was concerned it could be too slow per PR. I can look into moving it back to PR wise once the CI time is proved to be acceptable with essential ccache optimizations.